pkg/collector/corechecks/snmp/internal/checkconfig/config_test.go

```diff
@@ -2219,7 +2219,14 @@ func TestCheckConfig_Copy(t *testing.T) {
 	assert.NotSame(t, config.ProfileTags, configCopy.ProfileTags)
 	assert.NotSame(t, config.ExtraTags, configCopy.ExtraTags)
 	assert.NotSame(t, config.InstanceTags, configCopy.InstanceTags)
-	assert.NotSame(t, config.DeviceIDTags, configCopy.DeviceIDTags)
```

Comment:
Why is this assertion removed?

Reply:
@TCheruy It's a typo :) added back here 9713891e46